.github/container/Dockerfile.base

```diff
@@ -17,13 +18,28 @@ RUN apt-get update && \
         git \
         lld \
         vim \
+        bat \
+        curl \
+        git \
+        gnupg \
+        rsync \
         python-is-python3 \
         python3-pip \
+        liblzma-dev \
         wget \
     && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/*
-RUN pip install --upgrade --no-cache-dir pip
+RUN <<"EOF" bash -ex
+git config --global user.name "JAX Toolbox"
+git config --global user.email "jax@nvidia.com"
```

Comment:
There was some feedback earlier about how this could silently have commits become authored by the jax@nvidia.com user, which caused some issues during CLA presubmits. I reverted it for rosetta here: https://github.com/NVIDIA/JAX-Toolbox/pull/370. What do you think about reverting it here after it's no longer needed?

Reply:
What is the exact CLA issue? Can we use some other username/email that do not annoy the CLA presubmits?

Reply:
TJ was using our containers to commit code and this global username/email got used to author the commits. When a PR was made, google's CLA bot flagged it as a developer who hadn't signed a CLA.

Reply:
It's my opinion that we should remove the global username/email since this is kind of a silent failure since the end user doesn't get alerted that their commits are authored by jax@nvidia.com

Reply:
I see. So the container should not ship information that are user-specific. Makes sense.